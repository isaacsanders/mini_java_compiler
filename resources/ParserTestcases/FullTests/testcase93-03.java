 
 class java{
	public static void main(String[] args){
		boolean javaIsCool = true;
		boolean pythonIsCool = false;
		
		if(javaIsCool & pythonIsCool){
			System.out.print(1.0);		
		}
		
		derek = 1;
		theWorld = 0;
		
		if(derek > the World){
			string peaceOnEarth = true;
		}
		
		doNotDoThis
		
		/ I was almost a comment
		/*
		uh oh...
		
	
	}
 
 }