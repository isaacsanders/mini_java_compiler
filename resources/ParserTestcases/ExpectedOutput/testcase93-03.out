StmtList ::= 
Type ::= BOOLEAN
LiteralExpr ::= TRUE
CallExpr ::= LiteralExpr
UnaryExpr ::= CallExpr
Expr ::= UnaryExpr
Stmt ::= Type ID GETS Expr SEMI
StmtList ::= StmtList Stmt
Type ::= BOOLEAN
LiteralExpr ::= FALSE
CallExpr ::= LiteralExpr
UnaryExpr ::= CallExpr
Expr ::= UnaryExpr
Stmt ::= Type ID GETS Expr SEMI
StmtList ::= StmtList Stmt
Unmatched character '&' at line 7 and column 17


ERROR in line 7 and column 19: Syntax error

Error recovery to next semicolon.
StmtList ::= StmtList Stmt
LiteralExpr ::= INTEGER
CallExpr ::= LiteralExpr
UnaryExpr ::= CallExpr
Expr ::= UnaryExpr
Stmt ::= ID GETS Expr SEMI
StmtList ::= StmtList Stmt
LiteralExpr ::= ID
CallExpr ::= LiteralExpr
UnaryExpr ::= CallExpr
Expr ::= UnaryExpr


ERROR in line 14 and column 18: Syntax error



ERROR in line 27 and column 3: Couldn't repair and continue parse

Exception processing file: Can't recover from previous error(s)
